main.go

```diff
@@ -254,6 +255,14 @@ func InitFlags(fs *pflag.FlagSet) {
 		"Use deprecated infrastructure machine naming")
 	_ = fs.MarkDeprecated("use-deprecated-infra-machine-naming", "This flag will be removed in v1.9.")
 
+	fs.StringArrayVar(&syncMachineLabels, "sync-machine-labels", []string{
+		"node-role.kubernetes.io",
```

Comment:
is there a use case to disable these ones? i.e should the flag always be additive over the core ones supported?

Reply:
I do not have a strong preference i this case. From the original proposal it looks like the default labels were chosen because the they were identified as the recommended set of labels to sync down. 
I am +1 if we should do additive to default list.

Reply:
do we know if some users might prefer a more granular setup and have different labels for different clusters?

Reply:
Let's give sane defaults, if users want to disable or only allow a subnet, it should be an open choice imo

Reply:
Updated to include defaults and made the behavior to be additive to always sync the default set of labels.